src/schema/nodes/figure_element.ts

```diff
@@ -35,6 +38,7 @@ export const figureElement: NodeSpec = {
     id: { default: '' },
     attribution: { default: undefined },
     dataTracked: { default: null },
+    position: { default: undefined },
```

Comment:
Based on my comment in the ticket, let's change this to "type".

Reply:
We currently export the figure node as a figure element with  `fig-type = figure`, and the  listing node as a figure element with the `fig-type = listing`. We're not going to do that anymore?

Reply:
Yeah we're likely going to stop that. We're getting rid of listings at some point, so how we export them now doesn't matter much.

Reply:
Got it, done.